tasty-plutus/src/Test/Tasty/Plutus/TestData.hs

```diff
@@ -64,6 +64,7 @@ data TestData (p :: Purpose) where
   MintingTest ::
```

Comment:
This is an API change, which requires a major version bump, a CHANGELOG entry and a modified `@since`, as the constructor of `MintingTest` is exposed.